sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -45,4 +45,5 @@ export enum Role {
   StudentReissueMSFAA = "student-reissue-msfaa",
   StudentUpdateDisabilityStatus = "student-update-disability-status",
   StudentReverseScholasticStanding = "student-reverse-scholastic-standing",
+  StudentDeleteRestrictions = "student-delete-restrictions",
```

Comment:
AC can be updated with this role name to avoid confusion for QA or anyone looking later. 

<img width="637" height="49" alt="image" src="https://github.com/user-attachments/assets/d5a429d7-fc29-4a16-b6a1-7bf0940c8729" />

Reply:
I will request it to be updated.